Description: fix includes path & main()
Author: Nicolas HOUDELOT (nicolas@demosdebs.org)
Last-Update: 2020-04-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/main.c
+++ b/src/main.c
@@ -1338,7 +1338,7 @@
 void *__progname;
 /** \brief Object file starting point.
  */
-void _start()
+void main()
 {
 #endif
 #if defined(USE_LD)
--- a/src/record.c
+++ b/src/record.c
@@ -3,12 +3,12 @@
 #include <stdlib.h>
 #include <string.h>
 
-#include "png.h"
-#include "zlib.h"
+#include <png.h>
+#include <zlib.h>
 #ifdef __APPLE__
-#include "SDL/SDL_opengl.h"
+#include <SDL/SDL_opengl.h>
 #else
-#include "SDL_opengl.h"
+#include <SDL/SDL_opengl.h>
 #endif
 
 #include "record.h"
